app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.tsx

```diff
@@ -27,13 +27,20 @@ interface DocumentCaptureTroubleshootingOptionsProps {
    * Whether to display alternative options for verifying.
    */
   showAlternativeProofingOptions?: boolean;
```

Comment:
is anything with `alt` in the naming the b-version?

Reply:
No, the variants are referred to by their labels; if the config label of a variant is "LMNO" then it is represented in context as "in_person_variant_lmno".

Reply:
so alt is just anything that's not the default version we currently have?

Reply:
Sorry I misunderstood your question here. The `showAlternativeProofingOptions` value is not AB testing related; it's whether to show IPP. "Alternative" here means "not proofing online". This code / variable naming predates AB testing.

Reply:
ahhh got it!